src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -0,0 +1,157 @@
+                                                                           SortImports,
+                                                                           IDEDiagnosticIds.UseImplicitTypeDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExplicitTypeDiagnosticId,
+                                                                           IDEDiagnosticIds.AddQualificationDiagnosticId,
+                                                                           IDEDiagnosticIds.RemoveQualificationDiagnosticId,
+                                                                           IDEDiagnosticIds.PreferBuiltInOrFrameworkTypeDiagnosticId,
+                                                                           IDEDiagnosticIds.AddBracesDiagnosticId,
+                                                                           IDEDiagnosticIds.AddAccessibilityModifiersDiagnosticId,
+                                                                           IDEDiagnosticIds.OrderModifiersDiagnosticId,
+                                                                           IDEDiagnosticIds.MakeFieldReadonlyDiagnosticId,
+                                                                           IDEDiagnosticIds.RemoveUnnecessaryCastDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForConstructorsDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForMethodsDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForConversionOperatorsDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForOperatorsDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForPropertiesDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForIndexersDiagnosticId,
+                                                                           IDEDiagnosticIds.UseExpressionBodyForAccessorsDiagnosticId,
+                                                                           IDEDiagnosticIds.InlineDeclarationDiagnosticId,
+                                                                           CSharpRemoveUnusedVariableCodeFixProvider.CS0168,
+                                                                           CSharpRemoveUnusedVariableCodeFixProvider.CS0219,
+                                                                           IDEDiagnosticIds.UseObjectInitializerDiagnosticId,
+                                                                           IDEDiagnosticIds.UseCollectionInitializerDiagnosticId);
```

Comment:
definitely not happy seeing this duplication show up in *another* place.

Reply:
➡️ We're removing the duplication in the Tools &rarr; Options pages as part of another pull request for 16.0 Preview 1.